assets/js/lib/constants.js

```diff
@@ -0,0 +1,11 @@
+let defaultCountryText = "India";
```

Comment:
Let's use `const` instead of `let` to define constants. This way, we can be sure that no one can mistakenly change the value in these variables.

Also, use upper case variables to make it very clear when a variable is a constant. For example: `DEFAULT_COUNTRY_TEXT`.

Reply:
will do. ty.